README.md

```diff
@@ -1,26 +1,40 @@
 
 # Tutorials
 
+
 ## An introduction to the use of TERRA REF data and software
 
-Many of these tutorials have complex dependencies. In addition, many datasets are very large. These tutorials use the National Data Service Labs Workbench (NDS Labs Workbench). The NDS Labs Workbench provides the following:
+This repository provides a set of tutorials that are divided by data types and use cases. 
+
+In the repository, you will find three folders that contain examples of how to access data:
+
+* traits
+* sensors
```

Comment:
Similarly, which sensors? The ones on the Field Scanner? Ones across the TERRA-REF portfolio? Can we provide a link to the sensor enumeration right here?